Transient dynamic tables (#8741)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/03-upgrading-to-v1.12.md

## Adapter-specific features and functionalities

### Snowflake

> - You can create Snowflake dynamic tables as transient (no [Fail-safe period](https://docs.snowflake.com/en/user-guide/data-failsafe)) by setting the [`transient`](/reference/resource-configs/snowflake-configs#transient-dynamic-tables) config on models. The [`snowflake_default_transient_dynamic_tables`](/reference/global-configs/snowflake-changes#the-snowflake_default_transient_dynamic_tables-flag) flag controls the default behavior when `transient` is not set. Set this flag to `True` to create dynamic tables as transient when `transient` is not configured on the model.

Reviewer: ```suggestion
- You can create Snowflake dynamic tables as transient (no [Fail-safe period](https://docs.snowflake.com/en/user-guide/data-failsafe)) by setting the [`transient`](/reference/resource-configs/snowflake-configs#transient-dynamic-tables) config on models. 

When `transient` is not set on a model, the [`snowflake_default_transient_dynamic_tables`](/reference/global-configs/snowflake-changes#the-snowflake_default_transient_dynamic_tables-flag) flag controls the default. Set this flag to `True` to make all dynamic tables transient by default.
```
